web/README.md

```diff
@@ -0,0 +1,418 @@
+
+## APIs
+
+These are the REST APIs and their details:
+
+### Connect
+
+`/connect` is a POST API which is used in case of direct connection. It takes driver config in request body, following is the json structure for the same.
+
+#### Method
+
+`POST`
+
+#### Request body
+
+Connection information along with driver name.
+
+Example
+
+```json
+{
+  "Driver": "postgres",
+  "Host": "localhost",
+  "Port": "5432",
+  "Database": "dbname",
+  "User": "user",
+  "Password": "password"
+}
+```
+
+#### Response body
+
+No response body is returned.
+
+### Convert
+
+(1) `/convert/infoschema` is a GET API followed by `/connect` API to convert using infoschema mode. It returns the schema conversion state in json format.
+
+#### Method
+
+`GET`
+
+#### Request body
+
+No request body is needed.
+
+#### Response body
+
+Conv struct in JSON format.
+
+(2) `/convert/dump` is a POST API which is used to convert schema from dump file. It takes dump config in request body , following is the json structure for the same.
+
+#### Method
+
+`POST`
+
+#### Request body
+
+Provide driver and path to dump file.
+
+Example
+
+```json
+{
+  "Driver": "postgres",
+  "Path": "/path/to/dumpFile"
+}
+```
+
+#### Response body
+
+Conv struct in JSON format.
+
+### DDL
+
+`/ddl` is a GET API which must be used after using conversion APIs (i.e, `/connect` or `/convert`). This API returns the DDL statements of the converted schema.
+
+#### Method
+
+`GET`
+
+#### Request body
+
+No request body is needed.
+
+#### Response body
+
+Table wise DDL statements.
+
+Example
+
+```json
+{
+  "Albums": "--\n-- Spanner schema for source table Albums\n--\nCREATE TABLE Albums (\n    SingerId INT64 NOT NULL, -- From: SingerId smallint(6)\n    AlbumId INT64 NOT NULL,  -- From: AlbumId smallint(6)\n    AlbumTitle STRING(50)    -- From: AlbumTitle varchar(50)\n) PRIMARY KEY (SingerId, AlbumId)"
+}
+```
+
+### Session
+
+(1) `/session` is a GET API which returns the schema conversion state in json format. It also create a file with suffix `.session.json` in the `frontend/` folder.
+
+#### Method
+
+`GET`
+
+#### Request body
+
+No request body is needed.
+
+#### Response body
+
+Session file info with driver name.
+
+Example
+
+```json
+{
+  "Driver": "postgres",
+  "FilePath": "/path/to/sessionFile",
+  "FileName": "sessionFile.session.json",
+  "CreatedAt": "2021-01-01T00:00:00.000000+00:00"
+}
+```
+
+(2) `/session/resume` is a POST API which can be used to resume the previous session, it requires the session file which can be generated using the above API. It takes following as request body.
+
+#### Method
+
+`POST`
+
+#### Request body
+
+Provide driver and session file path.
+
+Example
+
+```json
+{
+  "driver": "postgres",
+  "path": "/path/to/sessionFile",
+  "fileName": "sessionFile.session.json"
+}
+```
+
+#### Response body
+
+No response body is returned.
+
+### Summary
+
+`/summary` is a GET API which returns table wise summary of the conversion.
```

Comment:
Suggest rephrasing as:

`/summary` is a GET API which returns a table-by-table report of the conversion.

Reply:
updated statement